Explorer/Assets/DCL/WebRequests/Texture/GetTextureWebRequest.cs

```diff
@@ -45,15 +54,46 @@ public CreateTextureOp(TextureWrapMode wrapMode, FilterMode filterMode)
                 this.filterMode = filterMode;
             }
 
+            public async UniTask<OwnedTexture2D?> ExecuteAsync(GetTextureWebRequest webRequest, CancellationToken ct)
+            {
+                try
+                {
+                    using var request = webRequest.UnityWebRequest;
+                    var data = request.downloadHandler?.nativeData;
+
+                    if (data == null)
+                        return null;
+
+                    var result = await webRequest.texturesUnzip.TextureFromBytesAsync(AsPointer(data.Value), data.Value.Length, webRequest.textureType, ct).Timeout(TimeSpan.FromSeconds(15));
+
+                    if (result.Success == false)
+                    {
+                        ReportHub.LogError(ReportCategory.TEXTURES, $"CreateTextureOp: Error loading texture url: {webRequest.url} - {result}");
+                        return null;
+                    }
+
+                    var texture = result.Value.Texture;
+
+                    texture.wrapMode = wrapMode;
+                    texture.filterMode = filterMode;
+                    texture.SetDebugName(webRequest.url);
+                    ProfilingCounters.TexturesAmount.Value++;
+                    return result.Value;
+                }
+                catch (Exception e)
```

Comment:
In the whole flow simply throw exceptions instead of trying to propagate `null` downwards: all systems were designed to work with exceptions instead of `null`.
It will also make the code cleaner